k8s.gcr.io/images/k8s-staging-dns/images.yaml

```diff
@@ -442,6 +460,7 @@
     "sha256:bc60c7d8b48ec0d48cd844b6f6f3d84776a19d543ed0af2415335f3ecbc6487e": ["1.22.9"]
     "sha256:c3f4cff132268503232b5fee5a7af5007456965f78713c20cf33c9a7e23253f1": ["1.22.10"]
     "sha256:2113f50e50d78d70b2b6a938478c34668b694b0b16604758015ad537bd1029ba": ["1.22.11"]
+    "sha256:XXX": ["1.22.12"]
```

Comment:
could you fix that pls?

Reply:
ditto
https://github.com/kubernetes/k8s.io/pull/4301/commits/d5c14231925976c6e42f438d556e156c2738f8b3